src/txmempool.cpp

```diff
@@ -619,13 +614,12 @@ static void CheckInputsAndUpdateCoins(const CTransaction& tx, CCoinsViewCache& m
 
 void CTxMemPool::check(const CCoinsViewCache *pcoins) const
 {
-    LOCK(cs);
-    if (nCheckFrequency == 0)
-        return;
+    if (m_check_ratio == 0) return;
 
-    if (GetRand(std::numeric_limits<uint32_t>::max()) >= nCheckFrequency)
+    if (GetRand(m_check_ratio) >= 1)
```

Comment:
This can be joined with the line below (in the first commit)

Reply:
updated 👍